templates/controllers/wizard/fileUpload/form/submissionFileMetadataForm.tpl

```diff
@@ -29,19 +29,6 @@
 		{/fbvFormSection}
 	{/fbvFormArea}
 
-	{* Read-only meta-data *}
```

Comment:
As I understand it, this would be for editors to know at a glance that the file is suitable for print/high-resolution web publishing.